pydoctor/astutils.py

```diff
@@ -737,3 +740,178 @@ class Precedence(object):
 
 del _op_data, _index, _precedence_data, _symbol_data, _deprecated
 # This was part of the astor library for Python AST manipulation.
+
+
+# Part of the sphinx.pycode.parser module.
+# Copyright 2007-2020 by the Sphinx team, see AUTHORS.
+# BSD, see LICENSE for details.
+from token import DEDENT, INDENT, NAME, NEWLINE, NUMBER, OP, STRING
+from tokenize import COMMENT, generate_tokens, tok_name
+
+class Token:
+    """Better token wrapper for tokenize module."""
+
+    def __init__(self, kind: int, 
+                 value: str, 
+                 start: Tuple[int, int], 
+                 end: Tuple[int, int],
+                 source: str) -> None:
+        self.kind = kind
+        self.value = value
+        self.start = start
+        self.end = end
+        self.source = source #: Source line
+
+    def __eq__(self, other: Any) -> bool:
+        if isinstance(other, int):
+            return self.kind == other
+        elif isinstance(other, str):
+            return bool(self.value == other)
```

Comment:
Coverage gaps like this are one reason I hate vendoring stuff this way 😬 . Could the examples be expanded to cover these cases at least?